src/aboutus.html

```diff
@@ -6,22 +6,52 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<meta name="description" content="Update your description here" />
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Jura&display=swap" rel="stylesheet">
 	<title>About Us Page</title>
 </head>
-
-<body>
-	<div class="topnav">
-		<a href="index.html">home</a>
-		<a href="moodselector.html">mood selector</a>
-		<a href="results.html">your music</a>
-		<a class="active" href="aboutus.html">about us</a>
+<body class="aboutus">	
+	<nav>
+		<a href="index.html" class="reftoindex">home</a>
+		<a href="moodselector.html" class="reftomood">mood selector</a>
+		<a href="results.html" class="reftores">your music</a>
+		<a href="aboutus.html" class="reftoact">about us</a>
+	</nav>
+<div class = "projectbox-aboutus">
+	<h4 class = "project-aboutus">THE PROJECT</h4>
+	<p class = "projecttext-aboutus">
+		The goal of this website is to provide users with new music recommendations depending on their current mood.<br><br>
+		Our interactive site takes the search and scrolls out of discovering new music.<br><br>
+		Users will be able to choose between different moods for instantaneous recommendations.<br><br>
+		And as a result, if users like the website's recommendations, they will be able to add them to their Spotify playlists.
+	</p>
+	<div class = "meet-teambox">
+		<h4 class = "meetteamtext">MEET OUR TEAM</h4>
```

Comment:
If you keep this capitalized, a screen reader will read it as M E E T to the user (individual letters). I would keep this proper cased, and use CSS's `text-transform: uppercase;` to apply this style instead.